src/Components/AdvancedSearchResultDetailsList/AdvancedSearchResultDetailsList.stories.tsx

```diff
@@ -0,0 +1,53 @@
+import React from 'react';
```

Comment:
Could you move these files into the internal folder for AdvancedSearch? The ones I created in this PR: #663